internal/cloud/realm/logs.go

```diff
@@ -95,7 +95,11 @@ type logsResponse struct {
 func (c *client) Logs(groupID, appID string, opts LogsOptions) (Logs, error) {
 	query := map[string]string{}
 	if len(opts.Types) > 0 {
-		query[logsQueryType] = strings.Join(opts.Types, ",")
+		formattedTypes := make([]string, len(opts.Types))
```

Comment:
I think there may be a different fix for this, now that I look more closely here...

in [`logs/list.go`](https://github.com/10gen/realm-cli/blob/master/internal/commands/logs/list.go#L67), we should probably be using `cmd.inputs.logTypes()` as the value for `opts.Types` (rather than just `cmd.inputs.Types` directly)

then I think this method remains unchanged, it would be nice to add a test somewhere in the `logs` package that made sure we passed the expected, formatted strings as an arg to the (mocked) client method